Cargo.toml

```diff
@@ -1,41 +1,47 @@
 [package]
 name = "lemmy-ui-leptos"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
How should changing this version work since there's no one actually depending on this frontend at the moment?
cc @dessalines

Reply:
honestly just a bit of fun :)